frontendActions/src/main/resources/GitMacheteBundle.properties

```diff
@@ -0,0 +1,221 @@
+
+
+action.GitMachete.BasePullBranchAction.action-name=P_ull
+action.GitMachete.BasePullBranchAction.description-action-name=Pull (fast-forward only)
+action.GitMachete.BasePullBranchAction.task-title=Pulling...
+
+
+action.GitMachete.BasePushBranchAction.action-name=_Push
+action.GitMachete.BasePushBranchAction.description-action-name=Push
+
+
+action.GitMachete.BaseRebaseBranchOntoParentAction.description=Rebase ''{0}'' onto ''{1}''
+action.GitMachete.BaseRebaseBranchOntoParentAction.description.disabled.merged=Cannot rebase merged branch ''{0}''
+action.GitMachete.BaseRebaseBranchOntoParentAction.description.disabled.repository.status=Cannot rebase ''{0}''
+action.GitMachete.BaseRebaseBranchOntoParentAction.description.disabled.repository.unknown-state=Cannot rebase due to unknown repository state
+action.GitMachete.BaseRebaseBranchOntoParentAction.description.disabled.root-branch=Root branch ''{0}'' cannot be rebased
+action.GitMachete.BaseRebaseBranchOntoParentAction.description.repository.state.detached-head=in the detached head state
+action.GitMachete.BaseRebaseBranchOntoParentAction.description.repository.state.ongoing.cherry-pick=during an ongoing cherry-pick
+action.GitMachete.BaseRebaseBranchOntoParentAction.description.repository.state.ongoing.merge=during an ongoing merge
+action.GitMachete.BaseRebaseBranchOntoParentAction.description.repository.state.ongoing.rebase=during an ongoing rebase
+action.GitMachete.BaseRebaseBranchOntoParentAction.description.repository.state.ongoing.revert=during an ongoing revert
+action.GitMachete.BaseRebaseBranchOntoParentAction.hook.task-title=Running machete-pre-rebase hook
+action.GitMachete.BaseRebaseBranchOntoParentAction.notification.abort=Rebase aborted
+action.GitMachete.BaseRebaseBranchOntoParentAction.notification.fail=Rebase failed
+action.GitMachete.BaseRebaseBranchOntoParentAction.task-title=Rebasing
+action.GitMachete.BaseRebaseBranchOntoParentAction.text=_Rebase Branch onto Parent
+
+
+# used indirectly within BaseResetBranchToRemoteAction
+action.GitMachete.BaseResetBranchToRemoteAction.description.enabled={0} branch ''{1}'' to its remote tracking branch
+action.GitMachete.BaseResetBranchToRemoteAction.action-name=Re_set to Remote
+action.GitMachete.BaseResetBranchToRemoteAction.description-action-name=Reset to remote
+action.GitMachete.BaseResetBranchToRemoteAction.notification.success=Branch ''{0}'' reset to remote
+action.GitMachete.BaseResetBranchToRemoteAction.notification.title=Resetting
+action.GitMachete.BaseResetBranchToRemoteAction.task-title=Resetting...
+
+
+action.GitMachete.BaseSlideInBranchBelowAction.description=Slide in branch below ''{0}''
+action.GitMachete.BaseSlideInBranchBelowAction.description.disabled.no-parent=Slide in disabled due to undefined parent branch
+action.GitMachete.BaseSlideInBranchBelowAction.dialog.create-new-branch.title=Create New Branch Below {0}
```

Comment:
This `Below` is unclear to me: it can suggest that branch will actually be under selected branch in graph but will be siblings of the selected branch, not its child. So maybe something like `Create New Child Branch` would be more accurate.

Reply:
A usual new branch dialog:
![image](https://user-images.githubusercontent.com/19799111/87804994-f3b8bd00-c854-11ea-8b48-f3e421dad02c.png)

I would rather keep this title (actually fix it, `s/Below/From/`).
- this is not OUR dialog; let's make it's title consistent 
- we have OUR dialog now that refers to slide in and machete logic; there is no need to reflect this reference here
- the idea in the current approach is to simply delegate branch creation to `git4idea` (if needed)

Reply:
Ouh, my fault. Not that line. I mean line 61 (action name and description)

Reply:
_slack discussion summary_

leaving as it is since `child` would in indicate that the branch is a child (but may be root)